doc/developer-guide/developer-guide.rst

```diff
@@ -21,23 +21,24 @@ Introduction
 The primary services that the CCArchiver interacts with are the Forwarder 
 (dm_Forwarder) and the ComCam Controller.  The Forwarder retrieves and assembles
 images. The ComCam Controller presents where files should be deposited and
-stages files where the Data Backbone (DBB) and the ComCam Observatory
-Operations Data Service (OODS) can act on them.  These services communicate
+stages files where the `Data Backbone https://dmtn-122.lsst.io` (DBB) and 
```

Comment:
These links aren't formatted correctly.
see:
https://dm-ccarchiver.lsst.io/v/DM-25960/developer-guide/developer-guide.html#introduction
The format should be:
```suggestion
stages files where the `Data Backbone <https://dmtn-122.lsst.io>`_ (DBB) and 
```

Reply:
Fixed